dataflow/src/main/java/org/checkerframework/dataflow/analysis/AbstractAnalysis.java

```diff
@@ -418,6 +419,20 @@ protected void addToWorklist(Block b) {
         }
     }
 
+    public @Nullable UnaryTree getUnaryTreeForAssign(AssignmentNode node) {
```

Comment:
1. add document in dataflow manual & source code
2. consider bi-directional Map
3. IdentityHashMap ?

Reply:
3 is related to 2. There is a https://github.com/google/guava/blob/master/guava/src/com/google/common/collect/HashBiMap.java but I didn't see a IdentityHashBiMap.
And these are related to the `cfg.getUnaryAssignNodeLookup();` call and how to simplify this method.